js/plates-model/model.ts

```diff
@@ -334,6 +334,9 @@ export default class Model {
                 props.age = neighbor.age;
                 // When continent is being stretched, move field marker to the new field to emphasise this effect.
                 props.marked = neighbor.marked;
+                // `blockFaulting` value doesn't have physical meaning, but it's used to determine its direction in the
+                // rendering code. 1e6 value is big enough to cover all the visible fields in the cross-section.
+                props.blockFaulting = (neighbor.blockFaulting ?? 1e6) - 1;
```

Comment:
`1e6 - 1` looks a little odd here, but just to confirm my understanding, the actual number doesn't matter since the only use for this value is the comparison down below, so the only thing that matters is which value is the relative value between neighbors.

Reply:
> the only thing that matters is the relative value between neighbors.

Yes, exactly. I've used 1e6, as it's pretty safe we won't have more than 1 million fields in a row :wink: Each next field will have a value decreased by 1. And even if we ever reach 0 for whatever reason, it'll actually automatically restart counting and it won't have big implications.